### PR TITLE
Proposed fix for release bug #2647

### PR DESCRIPTION
**Pull request by kylelang**
This PR should fix release bug [#2647](https://github.com/jasp-stats/jasp-test-release/issues/2647).

The original issue is caused by a limitation of JAGS. The JAGS implementation of the beta distribution requires strictly positive shape parameters. This PR attempts to enforce this restriction in two ways:

1. Imposing a small positive minimum value on the relevant QML fields.
2. Checking for non-positive values in the R code when parsing the analysis options (so the restriction is still honored when calling the R routines outside the JASP GUI).

### CODE REVIEW

**Comment by kylelang** [comment]
I need to fix the unit tests. This PR isn't actually ready for review.

**Comment by kylelang** [comment]
As far as I can tell, this PR should be ready to review/merge. The failed i18n check doesn't seem to relate to any of my changes. The failed Mac test may be a meaningful problem, but I don't have access to a Mac on which to debug the issue.

File: inst/qml/LSbinaryclassification.qml [hunk 1]
```diff
@@ -46,17 +46,73 @@ Form {
 		visible: inputType.value === "uncertainEstimates" || inputType.value === "data"
 		columns: 3
 		title: qsTr("Priors")
+
 		Text{ text: qsTr("Prevalence") }
-		FormulaField { name: "priorPrevalenceAlpha";  label: qsTr("~ Beta(α = ");		afterLabel: ",";		min: 0; defaultValue: "1"	; fieldWidth: 65}
-		FormulaField { name: "priorPrevalenceBeta";   label: "β = ";								afterLabel: qsTr(")");	min: 0; defaultValue: "9"; fieldWidth: 65	}
+
+		FormulaField
```
Comment (Kucharssim) [inline inst/qml/LSbinaryclassification.qml:52]: I think we could just set `inclusive` to false (https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-qml-guide.md#formulafield) to disallow 0 exactly...

Comment (kylelang) [reply inst/qml/LSbinaryclassification.qml:52]: That's a much nicer solution. I should have read the docs more carefully.

File: inst/qml/LSbinaryclassification.qml [hunk 2]
```diff
@@ -46,17 +46,79 @@ Form {
...
+			fieldWidth:		65
+		}
+
+		FormulaField
+		{
+			name:			"priorPrevalenceBeta"
+			label:			"β = "
+			afterLabel:		qsTr(")");
+			min:			0
+			inclusive:		0
```
Comment (boutinb) [inline inst/qml/LSbinaryclassification.qml:69]:
```suggestion
			inclusive:		false
```

File: inst/qml/LSbinaryclassification.qml [hunk 3]
```diff
@@ -46,17 +46,79 @@ Form {
...
-		FormulaField { name: "priorPrevalenceAlpha";  label: qsTr("~ Beta(α = ");		afterLabel: ",";		min: 0; defaultValue: "1"	; fieldWidth: 65}
-		FormulaField { name: "priorPrevalenceBeta";   label: "β = ";								afterLabel: qsTr(")");	min: 0; defaultValue: "9"; fieldWidth: 65	}
+
+		FormulaField
+		{
+			name:			"priorPrevalenceAlpha"
+			label:			qsTr("~ Beta(α = ")
+			afterLabel:		","
+			min:			0
+			inclusive:		0
```
Comment (boutinb) [inline inst/qml/LSbinaryclassification.qml:58]:
```suggestion
			inclusive:		false
```

File: inst/qml/LSbinaryclassification.qml [hunk 4]
```diff
@@ -46,17 +46,79 @@ Form {
...
-		FormulaField { name: "priorSensitivityAlpha"; label: qsTr("~ Beta(α = ");	afterLabel: ",";		min: 0; defaultValue: "8"; fieldWidth: 65	}
-		FormulaField { name: "priorSensitivityBeta";  label: "β = ";								afterLabel: qsTr(")");	min: 0; defaultValue: "2"; fieldWidth: 65	}
+
+		FormulaField
+		{
+			name:			"priorSensitivityAlpha"
+			label:			qsTr("~ Beta(α = ")
+			afterLabel:		","
+			min:			0
+			inclusive:		0
```
Comment (boutinb) [inline inst/qml/LSbinaryclassification.qml:82]:
```suggestion
			inclusive:		false
```

File: inst/qml/LSbinaryclassification.qml [hunk 5]
```diff
@@ -46,17 +46,79 @@ Form {
...
+			fieldWidth:		65
+		}
+
+		FormulaField
+		{
+			name:			"priorSensitivityBeta"
+			label:			"β = "
+			afterLabel:		qsTr(")")
+			min:			0
+			inclusive:		0
```
Comment (boutinb) [inline inst/qml/LSbinaryclassification.qml:93]:
```suggestion
			inclusive:		false
```

File: inst/qml/LSbinaryclassification.qml [hunk 6]
```diff
@@ -46,17 +46,79 @@ Form {
...
-		FormulaField { name: "priorSpecificityAlpha"; label: qsTr("~ Beta(α = ");	afterLabel: ",";		min: 0; defaultValue: "8"; fieldWidth: 65	}
-		FormulaField { name: "priorSpecificityBeta";  label: "β = ";								afterLabel: qsTr(")");	min: 0; defaultValue: "2"; fieldWidth: 65	}
+
+		FormulaField
+		{
+			name:			"priorSpecificityAlpha"
+			label:			qsTr("~ Beta(α = ")
+			afterLabel:		","
+			min:			0
+			inclusive:		0
```
Comment (boutinb) [inline inst/qml/LSbinaryclassification.qml:106]:
```suggestion
			inclusive:		false
```

File: inst/qml/LSbinaryclassification.qml [hunk 7]
```diff
@@ -46,17 +46,79 @@ Form {
...
+			fieldWidth:		65
+		}
+
+		FormulaField
+		{
+			name:			"priorSpecificityBeta"
+			label:			"β = "
+			afterLabel:		qsTr(")")
+			min:			0
+			inclusive:		0
```
Comment (boutinb) [inline inst/qml/LSbinaryclassification.qml:117]:
```suggestion
			inclusive:		false
```

**Pull request merged**